extensions/wikia/UserPreferencesV2/UserPreferencesV2.class.php

```diff
@@ -147,10 +147,15 @@ static public function onGetPreferences( $user, &$defaultPreferences ) {
 			$defaultPreferences['enotifminoredits']['label-message'] = 'tog-enotifminoredits-v2';
 			$defaultPreferences = self::moveToEndOfArray( $defaultPreferences, 'enotifminoredits' );
 		}
-		if ( isset( $defaultPreferences['enotifdiscussions'] ) ) {
-			$defaultPreferences['enotifdiscussions']['section'] = 'emailv2/email-me-v2';
-			$defaultPreferences['enotifdiscussions']['label-message'] = 'tog-enotifdiscussions-v2';
-			$defaultPreferences = self::moveToEndOfArray( $defaultPreferences, 'enotifdiscussions' );
+		if ( isset( $defaultPreferences['enotifdiscussionsvotes'] ) ) {
+			$defaultPreferences['enotifdiscussionsvotes']['section'] = 'emailv2/email-me-v2';
+			$defaultPreferences['enotifdiscussionsvotes']['label-message'] = 'tog-enotifdiscussions-votes-v2';
+			$defaultPreferences = self::moveToEndOfArray( $defaultPreferences, 'enotifdiscussionsvotes' );
+		}
+		if ( isset( $defaultPreferences['enotifdiscussionsfollows'] ) ) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "enotifdiscussionsfollows" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)